components/menu/test/menu-item-mixin.test.js

```diff
@@ -1,76 +1,80 @@
-import { defineCE, expect, fixture, oneEvent } from '@brightspace-ui/testing';
+import { clickElem, defineCE, expect, fixture, oneEvent, sendKeysElem } from '@brightspace-ui/testing';
+import { createMessage } from '../../../mixins/property-required/property-required-mixin.js';
 import { LitElement } from 'lit';
 import { MenuItemMixin } from '../menu-item-mixin.js';
 
 const tag = defineCE(
-	class extends MenuItemMixin(LitElement) {}
+	class extends MenuItemMixin(LitElement) {
+		render() { return this.text; }
```

Comment:
We need to render something, otherwise it has no width/height and can't be clicked.